autopush_rs/src/client.rs

```diff
@@ -690,9 +689,12 @@ where
         } else if all_acked && webpush.flags.check {
             transition!(CheckStorage { data });
         } else if all_acked && webpush.flags.rotate_message_table {
-            let response = data.srv.migrate_user(
-                webpush.uaid.simple().to_string(),
-                webpush.message_month.clone(),
+            debug!("Triggering migration");
+            let response = data.srv.ddb.migrate_user(
+                &webpush.uaid,
+                &webpush.message_month,
+                &data.srv.opts.current_message_month,
```

Comment:
so opts.current_message_month is essentially hardcoded at startup in autopush_rs.

ddb.migrate_user doesn't need to return it, then. await_migrate_user can assign `webpush.message_month = data.srv.opts.current_message_month.clone();` instead

Reply:
That's true.